lib/aruba/platforms/windows_command_string.rb

```diff
@@ -16,7 +16,8 @@ def initialize(cmd)
 
       # Convert to array
       def to_a
-        Shellwords.split __getobj__
+        Shellwords.split( __getobj__.gsub('\\', 'ARUBA_TMP_PATHSEPARATOR') ).
```

Comment:
What about replacing `\` by `/` instead? Looks more obvious where the problem is.